shell/components/form/ResourceTabs/index.vue

```diff
@@ -262,7 +266,6 @@ export default {
     >
       <!-- namespaced: false given we don't want the default handling of namespaced resource (apply header filter)  -->
       <PaginatedResourceTable
-        v-if="selectedTab === 'events'"
```

Comment:
We may want to put this back in. When the table isn't present the count will not show. There are other examples of this.

Reply:
We can't display the counts on a tab unless we fetch the resource to know what the count is.

This particular code that I removed prevents us from fetching the resource until someone selects the tab.  Therefore the tab count would never be shown until the user has selected the tab. 

The tab count should help inform the user that the tab has something worth looking at before they select the tab. So in this scenario the count is useless since we wouldn't show the count until the user has already selected the tab.